src/android/CameraLauncher.java

```diff
@@ -335,7 +335,9 @@ private File createCaptureFile(int encodingType) {
      */
     private File createCaptureFile(int encodingType, String fileName) {
         if (fileName.isEmpty()) {
-            fileName = ".Pic";
+            //creating timestamps for empty filename to differentiate between multiple files selected via gallery
+            String timeStamp = new SimpleDateFormat(TIME_FORMAT).format(new Date());
+            fileName = timeStamp;
```

Comment:
```suggestion
            // Append the current time in milliseconds for empty filename to differentiate between multiple files selected via gallery
            fileName = ".Pic-" + System.currentTimeMillis();
```

IMO, I would suggest the above.

What you have might work to a certain point.

The `TIME_FORMAT`'s lowest placement is in seconds. 

Could someone take burst shots and therefore have many photos per second. Using `System.currentTimeMillis` might help out a bit since it's now in milliseconds.

Example filename with the above code would be: `.Pic-1647402802991`

I would not recommend updating the value of `TIME_FORMAT`. If changed, then the PR becomes a major breaking PR.

To keep or not to keep the `".Pic-"` prefix is questionable? If it is kept as a prefix, then it could be used as a keyword filter when grabbing all files with the `.Pic` prefixed. I don't know if this is necessary, but I also don't know what other files exist in the same directory.

What do you think?